cv/lane_detection/src/line_fitting.py

```diff
@@ -28,13 +28,11 @@ def lane_fitting(points):
     x_width = abs(sorted_points_x[-1][1] - sorted_points_x[0][1])
     y_width = abs(sorted_points_y[-1][0] - sorted_points_y[0][0])
 
-    if (x_width < 15) or (y_width < 15):    # Hard-coded parameter, update maybe
+    if ((x_width > 0.95*y_width) and (x_width < 1.05*y_width) and (x_width < 20 or y_width < 20)) or len(points) < 200:    # Hard-coded parameter, update maybe
```

Comment:
Not readable if statement but it's fine if you want to keep it

Also delete the hardcoded parameter, and consider making this into a function that with a descriptive name (e.g. `is_in_roi`) that tells us what it does (but feel free to resolve since it's fine)

Reply:
this is actively being developed, will be pushed again by Jason